src/styles/index.css

```
@@ -1,18 +1,102 @@
+  font-weight: 100;
+  font-style: normal;
 }
+
+@font-face {
+  font-family: "Kanit";
+  src: url("../src/assets/fonts/Kanit-SemiBold.ttf") format("truetype");
+  font-weight: 600;
+  font-style: normal;
+}
+
 @import "tailwindcss";
```

Biome (1.9.4)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)